docs/docs/software-architecture.md

```diff
@@ -167,15 +166,15 @@ The user interacts with the Swapchain UI without signing-up or authenticating an
 System:   
 After the system receives the proposal submitted by the user a swap request is carried out using an HTLC. Therefore, the signatures are verified so that the private key matches the public key for the swap to take place. As soon as all required conditions are met it relays back and responds to the system to successfully transfer the funds. The involved users will receive a notification if the transaction was successful or not. In case the transaction fails, the users will receive a notification that they will be receiving the refund.
 
-
+   
 ## 8. Size and Performance
 
 The chosen software architecture supports the following requirements:
 
 1. The system shall support at least two simultaneous users at a time.
 2. The system shall be able to complete an ACCS transaction within 5 minutes once all HTLC requirements are met.
```

Comment:
guess that depends pretty much on how fast the blocks are mined, if you just missed on block by a bit you might have to wait longer. So I dont know if we can really give any specific time in which we assure to carry out the swap